cantools/database/can/formats/dbc.py

```diff
@@ -587,6 +587,10 @@ def _need_startval_def(database):
                for m in database.messages
                for s in m.signals)
 
+def _need_cycletime_def(database):
+    return any(m.cycle_time is not None
```

Comment:
Could you add an explanatory comment here? Otherwise this looks good to me.

Reply:
Oh, I figured it was similar enough to the function above it (`_need_startval_def`) to not need one either!

How about
```python
# If the user has added cycle times to a database which didn't start with them,
# we need to add the global attribute definition so the output DBC is valid
```

Reply:
Yes, that would be fine. Sorry for the late response